resources/gather-deploy-images.sh

```diff
@@ -24,6 +24,12 @@ function get-images-per-env() {
 	      # don't check images that didn't change between the current revision and the target branch
 	      continue
 	    fi
+
+	    # Workaround for RHTAPBUGS-1284
+	    if [[ "$image" =~ "quay.io/redhat-appstudio/dance-bootstrap-app" ]]; then
```

Comment:
what if we move this out of `if [ -n "$TARGET_BRANCH" ];` in case the pipeline runs on main, not in PR?

Reply:
Maybe but I'm not sure. I'm not clear on when/if the promote pipeline would ever run on main branch, since I think it's main purpose is to gate a merge to main.

I'll make a second PR anyway, but let's seek expert advice before merging either of them.

Reply:
Alternative PR: https://github.com/redhat-appstudio/tssc-sample-jenkins/pull/5

Reply:
Thinking about it some more I guess #5 is fine, right? There's probably no scenario where we do want EC to verify dance-bootstrap-app.